packages/@ember/-internals/glimmer/lib/renderer.ts

```diff
@@ -95,7 +96,23 @@ class RootState {
       alwaysRevalidate: false,
     });
 
-    this.render = () => {
+    let enhanceError;
+
+    if (DEBUG) {
+      enhanceError = (fn: () => void) => {
+        return () => {
+          try {
+            fn();
+          } catch (e) {
```

Comment:
See my comments in https://github.com/emberjs/ember.js/pull/18388#pullrequestreview-290151477. I worry that adding a try/catch here will make local debugging _worse_ overall (when devtools is open).